_content/news/_posts/2018-10-19-kerstkampen.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title:  "Basketbal voor de jeugd tijdens de kerstperiode"
+cover: /BasketLummen/Website/_content/albums/2018-10-13-Basket-Lummen-DSEA-Sint-Katelijne-Waver-DSEA/_thumbnails/IMG_6412.jpg
+date:   2018-10-25 20:00:00
+description: Basketbal voor de jeugd, ook tijdens de kerstperiode!.
+permalink: /news/vervang-dit-door-de-bestandsnaam-zonder-extensie/
+---
+
+Ook tijdens de kerstperiode zullen de jeugdspelers van Basket Lummen leerrijke ervaringen kunnen opdoen. Er is niet alleen het FROS-kamp, met coaches van Basket Lummen, maar ook het Bounce Camp Hans Vanwijn, een voormalig speler van Basket Lummen. Verder volgt er nog een evenement (Brecht)..... details...?
+
```

Comment:
Suggestie:

Ook tijdens de kersperiode kunnen onze jeugdspelers sportieve ervaring opdoen, er gaan namelijk een heleboel kampen en stages door tijdens de vakantie. Zo is er:
* Het FROS-kamp, met coaches van Basket Lummen
* Het Bouncecamp, georganiseerd door Hans Vanwijn, voormalig speler van onze club
* Onze eigen kerststage die door zal in de 2e week van de vakantie